### ISSUE TITLE
PY3: Cannot encrypt to a file

### ISSUE THREAD

**Issue by doktorstick** [isislovecruft/python-gnupg#93]
```python
import gnupg
import os

print ('gnupg version:', gnupg.__version__)

infile = os.path.expanduser ('~/usr/purernd.bin')
with open (infile, 'rb') as f:
    data = f.read()
    
kwargs = dict (passphrase='speedtest',
               symmetric=True,
               cipher_algo='AES256',
               armor=False,
               encrypt=False,
               output='/tmp/purernd.enc.gnupg')

gpg = gnupg.GPG (binary='/usr/bin/gpg')
r = gpg.encrypt (data, None, **kwargs)
print (r.ok)
```
```
gnupg version: 1.4.0
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-24-c5d667873aa8> in <module>()
     17 gpg = gnupg.GPG (binary='/usr/bin/gpg')
     18 t1 = time.perf_counter()
---> 19 r = gpg.encrypt (data, None, **kwargs)
     20 t2 = time.perf_counter()
     21 

/mnt/hgfs/work/branches/jayne/virts/jayne/lib/python3.4/site-packages/gnupg/gnupg.py in encrypt(self, data, *recipients, **kwargs)
    954         """
    955         stream = _make_binary_stream(data, self._encoding)
--> 956         result = self._encrypt(stream, recipients, **kwargs)
    957         stream.close()
    958         return result

/mnt/hgfs/work/branches/jayne/virts/jayne/lib/python3.4/site-packages/gnupg/_meta.py in _encrypt(self, data, recipients, default_key, passphrase, armor, encrypt, symmetric, always_trust, output, throw_keyids, hidden_recipients, cipher_algo, digest_algo, compress_algo)
    991             log.info("Writing encrypted output to file: %s" % output_filename)
    992             with open(output_filename, 'w+') as fh:
--> 993                 fh.write(result.data)
    994                 fh.flush()
    995                 log.info("Encrypted output written successfully.")

TypeError: must be str, not bytes
```

**Comment by isislovecruft** [comment]
This should be fixed in 2.0.1 (because #90 was merged).

**Comment by isislovecruft** [comment]
Okay… this wasn't fixed with the changes from #92; it got more broken for Python3.

**Comment by isislovecruft** [comment]
@doktorstick Thanks for reporting this issue! I just re-ran your example with Python3.4 after merging the above commit 43164fa into `develop` and it works just fine now, so it'll be fixed in python-gnupg>=2.0.1.
